tests/robot-tests/scripts/pipeline-run-rf-tests.sh

```diff
@@ -1,8 +1,10 @@
 #!/bin/bash
 
+[ "$#" -eq 1 ] || { echo "1 arguments required, $# given"; exit 1; }
```

Comment:
Might be worth saying in this message what argument it's expecting

Reply:
Changed